Dockerfile

```
@@ -56,3 +56,10 @@ RUN git clone --branch master https://github.com/pytorch/fairseq.git /fairseq
 WORKDIR /fairseq
 RUN pip install pytest pyyaml
 RUN pip install -e .
+
+FROM base as pytorch-lightning
+RUN git clone --branch 0.9.0 https://github.com/PyTorchLightning/pytorch-lightning /pytorch-lightning
+WORKDIR /pytorch-lightning
+RUN pip install --requirement ./requirements/base.txt --upgrade
+ENV LC_ALL=C.UTF-8
+RUN HOROVOD_BUILD_ARCH_FLAGS="-mfma" pip install -e ".[dev]" --global-option="--quiet" --upgrade
```

We'll need to wrap this in `sh -c` since `RUN` commands don't accept environment variables

```suggestion
RUN sh -c 'HOROVOD_BUILD_ARCH_FLAGS="-mfma" pip install -e ".[dev]" --global-option="--quiet" --upgrade'
```